### DISCUSSION TITLE
Project github.io 404

### DISCUSSION THREAD

**Discussion by Odomontois** [community/community#22918] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve created custom site here <a href="https://github.com/manatki/manatki.github.io" rel="nofollow noopener">https://github.com/manatki/manatki.github.io</a></p>
<p>First I’ve created in rush and repo name wasn’t correct, so I renamed it couple of times.</p>
<p>Now it seems to <a href="https://manatki.github.io" rel="nofollow noopener">https://manatki.github.io</a> still 404.</p>
<p>What could be the problem?</p>

**Comment by Odomontois** [comment, marked as answer]
<p>Ok. It’s appeared now</p>

**Comment by ulisigi** [comment]
<p>Hey <a class="mention" href="/odomontois">@odomontois</a></p>
<p>Your project probably just needed a little time for Pages to index your code before it deployed, hence the 404 you reported and that has since resolved. When you push updates, you shouldn’t experience the delay in the future. If you create a new project and deploy with Pages again, then the initial deployment will likely experience a similar delay and 404 until the indexing is completed. I hope this helps.</p>
<p>Happy coding!</p>
